js-miniapp-bridge/src/types/fetch.ts

```diff
@@ -0,0 +1,4 @@
+export type MiniAppRequestInit = Pick<
+  RequestInit,
+  'body' | 'headers' | 'method'
```

Comment:
We're starting out with support only for `GET` requests, so there's no need for `body` currently, right?

Reply:
If we don't have any security implications on providing POST method support, can we have the POST method implementation on the native-side with a simple string (raw-data, JSON) as a request body for the initial-phase?

Reply:
Hmm, I'm not sure - there aren't really any security concerns, but this will add more complexity on the native side, which we wanted to avoid for this initial implementation.

On the JS side, it's not very difficult to add support for POST requests, so maybe we can leave it out for now and then it will still be easy to add in JS when we decide to support it on the native side.